openlibrary/templates/lists/home.html

```diff
@@ -39,7 +39,9 @@ <h2>$_('Active Lists')</h2>
                             $ cover_url = cover and cover.url("S") or "/images/icons/avatar_book-sm.png"
                             <img src="$cover_url" alt=""/>
                         </div>
-                        $:_('%(listname)s </a> from <a href="%(key)s">%(owner)s</a>', key=owner.key, listname=list.name, owner=owner.displayname)
+                        %(listname)s
```

Comment:
I think @Yashs911 correctly spotted an error within this block which is resulting in erroneous behavior on http://staging.openlibrary.org/lists
![staging openlibrary org_lists](https://user-images.githubusercontent.com/978325/115385711-4b41db80-a18d-11eb-982f-41b10314c429.png)